index.js

```diff
@@ -107,14 +127,14 @@ NYC.prototype.addContent = function (filename, content) {
   }
 }
 
-NYC.prototype.shouldInstrumentFile = function (relFile, returnImmediately) {
-  // only instrument a file if it's not on the exclude list.
-  for (var i = 0, exclude; (exclude = this.exclude[i]) !== undefined; i++) {
-    if (exclude.test(relFile)) {
-      return false
-    }
+NYC.prototype.shouldInstrumentFile = function (filename, relFile) {
+  relFile = relFile.replace(/^\.\//, '') // remove leading './'.
+
+  if (this.include) {
+    return micromatch.any(filename, this.include) || micromatch.any(relFile, this.include)
```

Comment:
Had to do with how `minimatch` would choke on leading dots in a path. Not sure if `micromatch` has the same issue.